src/Sylius/Bundle/OrderBundle/Aggregator/AdjustmentsAggregatorInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\OrderBundle\Aggregator;
+
+use Sylius\Component\Order\Model\AdjustmentInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface AdjustmentsAggregatorInterface
+{
+    /**
+     * @param array|AdjustmentInterface[] $adjustments
```

Comment:
`AdjustmentInterface[]` is enough, the `array` part suggests that it accepts something else too.